### DISCUSSION TITLE
can I remove the collapsed button in the container

### DISCUSSION THREAD

**Discussion by Willia-Yang** [eclipse-sirius/sirius-desktop#490] [Q&A]
Hello，
question1: I created container, a collapsed button appears in the upper left corner, which I don't need, can I remove or hide it
![image](https://github.com/user-attachments/assets/3912c6fe-6263-4804-85e0-9c43fbbac753)
question2:The white area is an empty container, I'm just trying to make it take up part of the space on the right, but every time I click on it it shows its outer border, which is not what I want, is there any way that it won't show some black borders when I click on it
![image](https://github.com/user-attachments/assets/cff89218-3363-43ac-b687-903374fcec61)

**Comment by lak-proddev** [comment]
> question1: I created container, a collapsed button appears in the upper left corner, which I don't need, can I remove or hide it

Is there any configuration setting that will hide or disable it? We also need to hide it.

**Comment by lredor** [comment]
Hi,
There is no way to "configure" simply the removal of the collapse button.
This button is created in [RegionResizableEditPolicy.addCollapseHandle](https://github.com/eclipse-sirius/sirius-desktop/blob/master/plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/graphical/edit/policies/RegionResizableEditPolicy.java#L122) method.

You can probably create your own EditPart and adapt the corresponding EditPolicy to avoid the creation of this button. It will be probably the same for the selection handles of the "white container".

This [documentation ](https://eclipse.dev/sirius/doc/developer/extensions-provide_custom_style.html) is probably a good entry point.

Regards,

Laurent

**Reply by lak-proddev** [reply]
Thank you very much. I will check.
